.github/workflows/run-model-tests-1.yml

```diff
@@ -59,4 +59,5 @@ jobs:
 
       - name: run test
         run: |
-          TEST_SUBFOLDER=models docker compose run runtests
+          shopt -s nullglob
+          TEST_SUBFOLDER=$(ls tests/models/test_{a..l}*.py) docker compose run runtests
```

Comment:
I'm guessing this is because the model tests have been slow?  I think it may be because the NOIRLab server has been having problems.  It's been slow for the last several days; the problems we had last week with the gaia stars may have been all part of the same thing.

I made Issue #289 suggesting that we remove NOIRLab dependence from most of the tests, and hid the ones that still have it behind either an env var that skips them by default, or a mark that we expect it to fail sometimes.